manage.py

```diff
@@ -10,7 +10,7 @@
     if os.geteuid() == 0:
         exit("You can't run script as superuser")
 
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tttt.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tttt.settings.production")
```

Comment:
why did you change it? Tests failed in travis.
https://travis-ci.org/SatelliteQE/GreenTea/builds/127155228